wp-cli.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+class WPCOM_Compat_Command extends WPCOM_VIP_CLI_Command {
+
+	/**
+	 * Import protected embeds from WordPress.com
+	 *
+	 * # OPTIONS
+	 *
+	 * <file>
+	 * : The CSV file to import
+	 *
+	 * @subcommand import-protected-embeds
+	 */
+	function import_protected_embeds( $args ) {
+		list( $file ) = $args;
+
+		if ( ! file_exists( $file ) ) {
+			WP_CLI::error( 'Specified file does not exist' );
+		}
+
+		
+		$sql = 'CREATE TABLE `protected_embeds` ( ' .
+			'`id` bigint(20) unsigned NOT NULL AUTO_INCREMENT, ' .
+			'`embed_id` varchar(64) NOT NULL, ' .
+			'`src` varchar(255) NOT NULL, ' .
+			'`embed_group_id` varchar(64) NOT NULL, ' .
+			'`html` mediumtext, ' .
+			'`time_added` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP, ' .
+			'PRIMARY KEY (`id`), ' .
+			'UNIQUE KEY `embed_id` (`embed_id`) ' .
+			') ENGINE=InnoDB AUTO_INCREMENT=0 DEFAULT CHARSET=latin1';
```

Comment:
Probably need to pick the default charset here.

Reply:
Shouldn't it just match whatever charset the WP.com table uses?

Reply:
We export to CSV on wp.com, which should convert it to utf8. Then we import from CSV using $wpdb, which should convert it to whatever charset is set in wp-config. I'm worried that using a different charset than what's set in wp-config here will cause issues if the data is ever moved.